packages/communication-react/CHANGELOG.stable.md

```diff
@@ -1,9 +1,49 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Fri, 06 Oct 2023 19:59:45 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Nov 2023 16:44:11 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.10.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.10.0)
```

Comment:
Missed this in the grooming step. This should be '## [1.10.0](https://github.com/azure/communication-ui-library/tree/1.10.0)'